examples/invalid-journey.json

```diff
@@ -15,10 +16,15 @@
 			"type": "station",
 			"id": null,
 			"name": "Baz Main Station",
-			"coordinates": {
+			"location": {
+				"type": "loc",
```

Comment:
is `loc` valid as well? Or should this be `location` instead?

Reply:
After looking at my own PR again, I had also quickly stumbled over it. It's the invalid example though, so it's supposed not to be `location`. 😉

Reply:
Oh 🙈